lib/src/misc/bounds.dart

```diff
@@ -41,17 +43,17 @@ class Bounds<T extends num> {
   const Bounds.unsafe(this.min, this.max);
 
   /// Create a [Bounds] as bounding box of a list of points.
-  static Bounds<double> containing(Iterable<Point<double>> points) {
+  static Bounds<double> containing(Iterable<Offset> points) {
```

Comment:
This feels wrong. I think we should move `Bounds` to `Offset` completely, but otherwise we should not be mixing `Offset` and `Point`. I understand this is better for performance, but it is not great for consistency.

Reply:
I only migrated the interfaces that are `Bounds<double>`, as the intention is to convert entirely to using Offset instead of Points in the future. Reverting this back would actually require explicit conversion from Offset -> Point in multiple other parts of the code.

Reply:
Update, I migrated everything